pages/builders-challenge/index.html

```diff
@@ -239,11 +282,32 @@ <h3>Our Partners</h3>
     </div>
 </section>
 
+<section class="mzp-l-content mzp-t-content-xl c-challenge-event">
+    <table id="challenge-schedule" class="mzp-u-data-table">
+        <thead>
+            <tr>
+                <img src="/img/builders-challenge/event-agenda.png" alt="Event Agenda">
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
```

Comment:
Nit: This tag (and the closing tag) being outside the for loop seems to be creating an empty `<tr></tr>` at the top of the table.